converter/babeltrace.c

```diff
@@ -683,6 +709,12 @@ int main(int argc, char **argv)
 	if (opt_output_format)
 		strlower(opt_output_format);
 
+#ifdef ENABLE_DEBUGINFO
+	if (opt_debug_dir) {
+		debug_info_set_debug_dir(opt_debug_dir);
```

Comment:
`debug_info_set_debug_dir()` should be stubbed to have no effect if `ENABLE_DEBUGINFO` is not defined.